data/io.elementary.desktop.agent-geoclue2.desktop.in

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
-_Name=Location Dialog
-Exec=${EXECDIR}/pantheon-agent-geoclue2 
+Name=Location Dialog
+Exec=/usr/libexec/geoclue2-1-pantheon/pantheon-agent-geoclue2 
```

Comment:
Any reason for not replicating the functionality of CMake here and using some sort of configuration variable?

Reply:
Mainly just because it was less work not to and this path is [hardcoded in packaging already](https://github.com/elementary/pantheon-agent-geoclue2/blob/deb-packaging/debian/pantheon-agent-geoclue2.desktop#L4) anyways.